packages/angular/src/app/components/header/header.component.ts

```diff
@@ -9,6 +9,7 @@ import { DxToolbarModule } from 'devextreme-angular/ui/toolbar';
 import { Router } from '@angular/router';
 import { UserPanelModule } from '../user-panel/user-panel.component';
 import { AuthService, IUser } from '../../services';
+import {ThemeSwitcherModule} from "../theme-switcher/theme-switcher.component";
```

Comment:
Add indents

Reply:
Let's enforce eslint rules or prettier usage for such cases: indents, import sorting etc.
As a separate task of course.